features/step_definitions/active_admin_steps.rb

```diff
@@ -78,5 +78,15 @@
 
 Given("An admin exists {string} with {string} and super_admin {string}") do |email, password, state|
   state == 'true' ? condition = true : condition = false
-  FactoryGirl.create(:admin_user, email: email, password: password, super_admin: condition)
+  @restaurant_owner = FactoryGirl.create(:admin_user, email: email, password: password, super_admin: condition)
```

Comment:
The naming of this variable is only relevant if this step is used in the context of a restaurant owner. I think you are using it also when creating a super_user, right?

Reply:
No, we have that somewhere else, need some refactoring some steps to keep it DRY.